### PR TITLE
update weights

### PR DESCRIPTION
**Pull request by coderobe**
this PR will also be backported to the current release branch once reviewed & merged

### CODE REVIEW

**Comment by ggwpez** [comment]
Looks like only a few extrinsics changed on [swc](https://weights.tasty.limo/compare?old=5e5dce8aad3deae30f5c4d683d1f50995a5ce4d6&new=20b3523e9c8754917364b8e7d53a1bc2c8c552af&threshold=15&unit=time&repo=polkadot&path_pattern=**%2Fweights%2F*.rs&method=worst&ignore_errors=true#runtime/kusama/src/weights/pallet_staking.rs.get_npos_voters). Some random samples should still be checked manually to see if there are bugs in swc.  
I would consider every change <1µs as insignificant, since the `ExtrinsicBaseWeight` is about 80µs.
 
The only one that really asymptotically increased is the `get_npos_voters`.  
image
Issue https://github.com/paritytech/substrate/issues/11397 will then give exact numbers for the asymptotic behaviour, but in this case its correct since `s` has a maximum of 100 in the staking benchmarks.

**Review by ggwpez — APPROVED** [review]

File: runtime/kusama/src/weights/pallet_elections_phragmen.rs [hunk 1]
```diff
@@ -147,12 +147,10 @@ impl<T: frame_system::Config> pallet_elections_phragmen::WeightInfo for WeightIn
...
 	// Storage: Balances Locks (r:250 w:250)
 	// Storage: System Account (r:250 w:250)
-	fn clean_defunct_voters(v: u32, d: u32, ) -> Weight {
+	fn clean_defunct_voters(v: u32, _d: u32, ) -> Weight {
 		(0 as Weight)
-			// Standard Error: 42_000
-			.saturating_add((47_366_000 as Weight).saturating_mul(v as Weight))
 			// Standard Error: 40_000
-			.saturating_add((240_000 as Weight).saturating_mul(d as Weight))
+			.saturating_add((48_208_000 as Weight).saturating_mul(v as Weight))
```
Comment (EgorPopelyaev) [inline runtime/kusama/src/weights/pallet_elections_phragmen.rs:153]: Big jump up of weight value

File: runtime/polkadot/src/weights/pallet_utility.rs [hunk 2]
```diff
@@ -45,27 +45,24 @@ use sp_std::marker::PhantomData;
 pub struct WeightInfo<T>(PhantomData<T>);
 impl<T: frame_system::Config> pallet_utility::WeightInfo for WeightInfo<T> {
 	fn batch(c: u32, ) -> Weight {
-		(4_943_000 as Weight)
-			// Standard Error: 2_000
-			.saturating_add((3_798_000 as Weight).saturating_mul(c as Weight))
+		(11_289_000 as Weight)
```
Comment (EgorPopelyaev) [inline runtime/polkadot/src/weights/pallet_utility.rs:48]: ≈x2 jump

**Review by EgorPopelyaev — APPROVED** [review]

File: runtime/kusama/src/weights/pallet_elections_phragmen.rs (hunk 1 above)
Comment (ggwpez) [reply runtime/kusama/src/weights/pallet_elections_phragmen.rs:153]: The git diff is a bit confusing here, but parameter `v` is nearly the same as before.  
Paramete `d` got kicked out. This is only a very small increase.

File: runtime/polkadot/src/weights/pallet_utility.rs (hunk 2 above)
Comment (ggwpez) [reply runtime/polkadot/src/weights/pallet_utility.rs:48]: The constant component does not really matter here compared to the component `c`.  
`c` goes up to 1000 for this benchmark. SWC will soon take the correct slope maxima into account to simplify these comparisions.

**Pull request merged**